moveit_ros/moveit_servo/test/pose_tracking_test.test

```diff
@@ -15,7 +15,7 @@
     <param name="publish_frequency" type="double" value="50.0"/>
   </node>
 
-  <test pkg="moveit_servo" type="pose_tracking_test" test-name="pose_tracking_test" time-limit="60
+  <test pkg="moveit_servo" type="pose_tracking_test" test-name="pose_tracking_test" time-limit="300
```

Comment:
Is that really all that's needed? I'm a little concerned since the test should really not take that long, something might be blocking it that we are not aware of.

Reply:
I believe that the test timeout duration is not related to the CI issue we had encountered in #320. Unfortunately that doesn't leave us any further ahead than we were previously -- while we were able to get the CI pipeline for #320 succeeding after restarting the job, we don't have a conclusive answer about why it was failing.